web/js/notifications/wv.notifications.ui.js

```diff
@@ -58,6 +58,10 @@ wv.notifications.ui = wv.notifications.ui || function(models, config) {
    */
   var init = function() {
     var reactComponent, options, p, alertUser;
+    if(!wv.util.browser.localStorage) {
```

Comment:
@localjo disabled this module here if localstorage isn't present. As this module is pointless without it.